constitution.md

```diff
@@ -0,0 +1,196 @@
+If a candidate fails to receive a majority of votes, a run off election will be
+held within the top two candidates that received the most votes. Members
+interested in becoming an officer must meet the following academic requirement:
+2.2 cumulative G.P.A. The term of office will be one full year (when to when).
+All officers shall comprise the Executive Committee of the organization. The
+Executive Committee shall meet in addition to regular organization meetings.
+The Executive Committee shall appoint such committees that are needed to carry
+out organization goals. The officers and individual duties shall be:
+
+- President
+  - Preside over all meetings
+  - Represent organization on campus
+  - Ensure that the organization is operating in conformity with the standards
+  set forth by James Madison University and the Office of Student Activities &
+  Involvement
+  - Cosign organization checks with Treasurer
+  - Maintain communication with organization adviser
+
+- Vice-president
+  - Preside over meetings in the absence of the President
+  - Schedule meetings/events with appropriate University offices
+  - Coordinate organization promotion and publicity of events
+
+- Secretary
+  - Maintain an accurate record of all organization meetings and post for
+members
+  - Maintain membership directory
+  - Correspond when necessary with University administration and other
+recognized organizations
+
+- Treasurer
+  - Maintain accurate record of organization transactions
+  - Collect dues if required
+  - Develop organization budget and present to membership for ¾ vote
+  - Cosign organization checks with President
+  - Arrange fundraising opportunities for the organization
+  - Solicits additional funding if needed from the Student Government
+Association in conjunction with the President
+
+Officers may be removed from office by ½ vote of the other officers and ¾ of
+the general membership if actions are deemed inappropriate by the membership.
+The officer is permitted to speak before the Executive Committee and the
+general membership about the charges made concerning his/her performance. The
+officer is not permitted to participate in the deliberation of the Executive
+Committee regarding the charges.
+
+## Article V - Adviser
+
+The adviser shall be a full or part time faculty or staff member at James
+Madison University. The adviser will assume those responsibilities as outlined
+in this constitution and/or found in University Policy No. 3101 entitled
+"Faculty Advisers to Student Organizations." The adviser will be selected by a
+½ vote of the Executive Committee and then presented to the general membership
+for a ¾ vote. Advisers not fulfilling responsibilities or abiding by the
+organization’s purpose may be removed from the position by a ½ vote of the
+Executive Committee. (List basic expectations the organization has of the
+adviser)
+
+## Article VI - Meetings
```

Comment:
A lot of this section just really feels like it doesn't apply to us. Certainly Robert's Rules of Order would make our meetings interesting. And I think we'd have a hard time keeping a secretary if they had to get an email from everyone who would miss a particular week.

Reply:
We may want to make a stronger distinction between regular meetings and "business meetings," the latter of which have sort of happened a bit informally at the end of regular meetings in the past, and honestly it wouldn't hurt to be a little more formal about them. This would also alleviate the burden on the secretary by removing responsibility for the non-business part of each meeting.